ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -117,31 +117,55 @@ class LocalAssemblerData
         }
     }
 
-    Eigen::Map<const Eigen::RowVectorXd>
-    getShapeMatrix(const unsigned integration_point) const override
+    std::vector<double> getFlux(
+        MathLib::Point3d const& p,
+        std::vector<double> const& local_x) const override
+    {
+        // eval dNdx and invJ at p
+        using FemType =
+            NumLib::TemplateIsoparametric<ShapeFunction, ShapeMatricesType>;
+
+        FemType fe(*static_cast<const typename ShapeFunction::MeshElement*>(
+            &_element));
+
+        typename ShapeMatricesType::ShapeMatrices shape_matrices(
+            ShapeFunction::DIM, GlobalDim, ShapeFunction::NPOINTS);
+
+        fe.computeShapeFunctions(p.getCoords(), shape_matrices, GlobalDim);
```

Comment:
OK, here you need the Point3D. I guess it has to be in natural coordinates. Maybe that should be documented (and reflected in the argument name).